### PR TITLE
Fix/dref final report

### PR DESCRIPTION
**Pull request by k9845**
Addresses 
- https://github.com/IFRCGo/go-frontend/issues/1948
## Changes
 - Fix database fields in the final report

## Checklist 
Things that should succeed before merging.

- [x] Updated/ran unit tests
- [ ] Updated CHANGELOG.md

## Release

If there is a version update, make sure to tag the repository with the latest version.

### CODE REVIEW

File: dref/serializers.py [hunk 1]
```diff
@@ -249,8 +260,8 @@ def validate(self, data):
             })
         if self.instance and self.instance.is_published:
             raise serializers.ValidationError('Published Dref can\'t be changed. Please contact Admin')
-        if self.instance and DrefFinalReport.objects.filter(dref=self.instance).exists():
-            raise serializers.ValidationError('Field Report already generated for dref %s' % self.instance.id)
+        if self.instance and DrefFinalReport.objects.filter(dref=self.instance, is_published=True).exists():
+            raise serializers.ValidationError('Can\'t Update %s dref for publish Field Report' % self.instance.id)
```
Comment (thenav56) [inline dref/serializers.py:264]: ugettext

**Review by thenav56 — CHANGES_REQUESTED** [review]
Also, can we merge 5 migrations into one?

**Comment by frozenhelium** [comment]
@szabozoltan69  please do the honors!

**Comment by szabozoltan69** [comment]
Will be deployed ASAP (in 8 minutes).

**Comment by szabozoltan69** [comment]
@frozenhelium Deployed

**Pull request merged**